x-pack/plugins/apm/public/components/app/service_overview/service_overview_dependencies_table/index.tsx

```diff
@@ -182,7 +179,7 @@ export function ServiceOverviewDependenciesTable({ serviceName }: Props) {
           query: {
             start,
             end,
-            environment: environment || ENVIRONMENT_ALL.value,
+            environment,
```

Comment:
I like this change because what we were doing before seemed fishy but... Is it still correct?

Reply:
Yes because now we're doing this in what comes back from `useUrlParams`: https://github.com/smith/kibana/blob/33457aaf688c87ebf9d7e7101caa39608266bb20/x-pack/plugins/apm/public/context/url_params_context/resolve_url_params.ts#L70